### ISSUE TITLE
New running indicator style?

### ISSUE THREAD

**Issue by rsjudka** [micheleg/dash-to-dock#789]
Hello,

I've created my own running indicator class including some GTK+ effects and was wondering if this might be something worth including in the repo. Here is a preview of what it looks like:
![preview](https://github.com/rsjudka/dash-to-dock/blob/master/media/screenshot-gtk.png)

Currently, my implementation is just a little hack I threw together to customize it to my liking, but if anyone would like I could do a little refactoring and get this worthy for a pr

One thing I'm unsure about tho is whether this would be better as a single style (similar to Ciliora or Metro) or as a toggle in the indicator settings (like the "enable unity 7 like..." option)

**Comment by micheleg** [comment]
Hi, I would consider a clean implementation. Not sure either if it should be a general option or a specific style. A style would be more isoalted therefore easier to include, but you might end up duplicating code.

Can you describe the style implemented and maybe link to the code to better underatand what you are trying to achieve?

**Comment by rsjudka** [comment]
Of course. So for the style, I wanted to accent the focused application but done minimally. I took the idea from the default style on Ubuntu Budgie 18.04 (not sure if it's from the GTK+ theme or not) which basically takes the app's palette color and applies a gradient as the backlight.

For the code, I added the [RunningIndicatorGTK class](https://github.com/rsjudka/dash-to-dock/blob/master/appIconIndicators.js#L667) which is essentially a tweaked version of the RunningIndicatorDots which overrides the _enableBackLight and _updateCounterClass functions (and changes some things in the init).

My original plan was to extend it from RunningIndicatorBase, but I couldn't figure out how to override super super class functions.
